debian/changelog

```diff
@@ -1,5 +1,12 @@
 kano-settings (4.0.2-0) unstable; urgency=low
```

Comment:
Increase version?

Reply:
You're right! I caught that one in this other PR: https://github.com/KanoComputing/kano-settings/pull/463/files

Reply:
Oh! Great!